### DISCUSSION TITLE
可以通过权限设置指定用户无法看到指定环境env配置吗

### DISCUSSION THREAD

**Discussion by yang0009** [apolloconfig/apollo#4694] [Q&A]
能通过权限设置让指定用户看不到指定环境的namespace配置吗，注意是无法看到，而不是无法修改发布

**Comment by yang0009** [comment]
![image](https://user-images.githubusercontent.com/47616231/210754917-44568d55-4578-40d0-a400-66246d656261.png)
按照文档,我配置成这样,不生效

**Reply by nobodyiam** [reply]
Here is the [documentation](https://www.apolloconfig.com/#/zh/usage/apollo-user-guide?id=_61-%e9%85%8d%e7%bd%ae%e6%9f%a5%e7%9c%8b%e6%9d%83%e9%99%90). Only private namespaces are affected.
